src/commands/deploy/IDeployBaseContext.ts

```diff
@@ -14,6 +14,7 @@ export interface IDeployBaseContext extends ISubscriptionActionContext {
     targetContainer?: ContainerAppModel;
 
     imageSource?: ImageSourceValues;
+    buildType?: string;
```

Comment:
This should probably be a stronger type than just `string`. Either an enum or string union.